lisa/tools/mkfs.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from typing import Any, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+
+
+class Mkfs(Tool):
+    __EXIST_FILE_SYSTEM_PATTERN = re.compile(
+        r".*appears to contain an existing filesystem", re.MULTILINE
+    )
+
+    @property
+    def command(self) -> str:
+        return self._command
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def mkfs(self, disk: str, command: str) -> None:
+        self._command = command
+        cmd_result = self.node.execute(
+            f"echo y | {self.command} {disk}", shell=True, sudo=True
+        )
+        if self.__EXIST_FILE_SYSTEM_PATTERN.match(cmd_result.stdout):
+            cmd_result = self.node.execute(
+                f"echo y | {self.command} -f {disk}", shell=True, sudo=True
+            )
+        if 0 != cmd_result.exit_code:
```

Comment:
Any logic is used to validate the exit_code, it should log the exit code too.

Reply:
We should at least log what is going on here, since this are destructive ops going on.